src/integration/render.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { JSDOM } from 'jsdom';
+import { Helmet } from 'react-helmet';
+import { renderToString, renderToStaticMarkup } from 'react-dom/server';
+import { render } from '@testing-library/react';
+import { within } from '@testing-library/dom';
+import { MemoryRouter } from 'react-router-dom';
+import { matchPath } from 'react-router';
+import routes from '../app/routes';
+import App from '#containers/App/App'; // eslint-disable-line import/no-named-as-default
+import DocumentComponent from '../server/Document/component';
+
+/* getByText can only select text inside of one dom element and you will see error:
+ * "Unable to find an element with the text: 'example'. This could be because the text is broken up by multiple elements."
+ * Use getByTextSpecial to select text that spans multiple dom elements e.g. <h1>This is a <span>headline</span></h1>
+ */
+const getByTextSpecial = getByText => text =>
```

Comment:
```suggestion
const getByTextMultiElement = getByText => text =>
```
Just a suggestion of a different name, may be a bit more descriptive?

Reply:
Much better naming, thanks!